src/test/java/tests/RepoIDTest.java

```diff
@@ -0,0 +1,81 @@
+package tests;
+
+import backend.RepoID;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RepoIDTest {
```

Comment:
Our convention (although it is broken in many places) is to use strict camel case. i.e. `RepoIdTest`